x-pack/plugins/threat_intelligence/public/modules/timeline/components/add_to_timeline/add_to_timeline.tsx

```diff
@@ -53,9 +53,10 @@ export const AddToTimeline: VFC<AddToTimelineProps> = ({ data, field, component,
   let value: string | null;
   if (typeof data === 'string') {
     value = data;
-  } else if (field === ComputedIndicatorFieldId.DisplayValue) {
-    field = displayField(data) || '';
-    value = displayValue(data);
+  } else if (field === ComputedIndicatorFieldId.DisplayName) {
+    const displayName = getDisplayName(data);
+    field = displayName[0];
+    value = displayName[1];
```

Comment:
using `[0]` and `[1]` isn't super clear... wouldn't it be better to have a obj returned instead of an array?

Reply:
good point, I made the change. I keep it as an array internally but returning the object from `getDisplaName`